lib/util/timestamp.js

```diff
@@ -0,0 +1,21 @@
+// Ensure that we have 2 places for each of the date segments.
+function padDate(segment) {
+  segment = segment.toString();
+  return segment[1] ? segment : `0${segment}`;
```

Comment:
Probably this can be used: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Reply:
I only moved this code from lib/migrate/MigrationGenerator.js to here, to not change the logic and possibly causes any problem.